### DISCUSSION TITLE
How can I change table's rows-per-page icon in bottom?

### DISCUSSION THREAD

**Discussion by bshafiei-ir** [quasarframework/quasar#13422] [General - Components / Directives / etc]
I'm using table component and md6 as icons.
As you can see in screenshot, It doesn't show as right dropdown icon, How can I change it to mdi-chevron-down?

![image](https://user-images.githubusercontent.com/19801131/168416260-1cd2ec84-5281-4245-86b9-5b4767f49359.png)

**Comment by metalsadman** [comment]
Be sure you set it https://quasar.dev/options/quasar-icon-sets#quasar-cli-way

```js
extras: [
  'mdi-v6'
],
framework: {
  iconSet: 'mdi-v6'
}
```
